### PR TITLE
Branchleven

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: TestWebAppProject/TestWebAppProject.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp1.1.1</TargetFramework>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: 1111111

File: TestWebAppProject/TestWebAppProject.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: here too

File: TestWebAppProject/TestWebAppProject.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   </PropertyGroup>
```
Comment (ghfvs-tester) [inline TestWebAppProject/TestWebAppProject.csproj]: comment

File: TestWebAppProject/TestWebAppProject.csproj (hunk 1 above)
Comment (meaghanlewis-tester) [reply TestWebAppProject/TestWebAppProject.csproj]: i think this version should be different

File: TestWebAppProject/TestWebAppProject.csproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
```
Comment (meaghanlewis-tester) [inline TestWebAppProject/TestWebAppProject.csproj]: question?

Comment (meaghanlewis-tester) [reply TestWebAppProject/TestWebAppProject.csproj]: blah blah blah

**Review by meaghanlewis-tester — CHANGES_REQUESTED** [review]

**Review by meaghanlewis-tester — APPROVED** [review]
lgtm

File: TestWebAppProject/TestWebAppProject.csproj [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:3]: property group

File: TestWebAppProject/TestWebAppProject.csproj [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp1.1.2</TargetFramework>
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:5]: new comment

File: TestWebAppProject/TestWebAppProject.csproj [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp1.1.2</TargetFramework>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:4]: new comment here

File: TestWebAppProject/TestWebAppProject.csproj [hunk 8]
```diff
@@ -1,7 +1,7 @@
...
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
   </PropertyGroup>
```
Comment (ghfvs-tester) [inline TestWebAppProject/TestWebAppProject.csproj:6]: hello

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — APPROVED** [review]

**Review by tierninho — COMMENTED** [review]
hhh﻿

**Review by tierninho — APPROVED** [review]

**Review by tierninho — COMMENTED** [review]
dsfsdf
﻿

**Review by tierninho — COMMENTED** [review]
change something﻿

**Pull request closed without merging**